source/shared/cpp/ObjectModel/BaseCardElement.h

```diff
@@ -23,6 +23,9 @@ class BaseCardElement
     virtual bool GetSeparator() const;
     virtual void SetSeparator(const bool value);
 
+    HeightType GetHeight() const;
```

Comment:
>GetHeight [](start = 15, length = 9)

Looks like the Android build is failing because this has the same name as an existing function on Image...

Reply:
The issue I'm seeing is that swig is not generating the function correctly, it should return a HeightType data type instead of the long, similar to GetSpacing or GetImageSize, I'm not sure how to fix this D:

---
In reply to: [184262303](https://github.com/Microsoft/AdaptiveCards/pull/1377#discussion_r184262303) [](ancestors = 184262303)

Reply:
Sorry, I get the issue now

---
In reply to: [184477639](https://github.com/Microsoft/AdaptiveCards/pull/1377#discussion_r184477639) [](ancestors = 184477639,184262303)